### ISSUE TITLE
Selecting across wbr element Bug 584141

### ISSUE THREAD

**Issue by nos111** [w3c/selection-api#113]
Currently selecting text that contains a wbr element will not cross the wbr element.

The new behavior will select cross the wbr element.

https://bugzilla.mozilla.org/show_bug.cgi?id=584141

**Comment by annevk** [comment]
In particular, if we ever define the overall model of selection better, this is likely the behavior we want to describe for `wbr`.

**Comment by rniwa** [comment]
Sorry, could you clarify how this will affect the selection API? Because the current doesn't specify precisely how user interaction would end up selecting content, this seems like UA dependent behavior. Or is your commentary about "define the overall model of selection" about defining precisely which nodes will be selected & when?

**Comment by annevk** [comment]
Given that there are interoperability issues because of this, I think it makes sense for the specification to define high-level selection operations (e.g., "select word", "select all") and their implications. Given how we organize other standards it seemed most natural to me that the model would be defined alongside the API.

**Comment by rniwa** [comment]
> Given that there are interoperability issues because of this, I think it makes sense for the specification to define high-level selection operations (e.g., "select word", "select all") and their implications. Given how we organize other standards it seemed most natural to me that the model would be defined alongside the API.

Could you elaborate more on what exactly you mean by "the model"? Like which nodes will be selected when extending selection?

**Comment by annevk** [comment]
Yeah, and which nodes are to be ignored and such for the purposes of determining boundaries.
